README.md

```diff
@@ -2,7 +2,7 @@
 
 [![Build Status](https://travis-ci.org/rust-analyzer/rust-analyzer.github.io.svg?branch=src)](https://travis-ci.org/rust-analyzer/rust-analyzer.github.io)
```

Comment:
We should remove this.

Reply:
Ah ye we should, wanted to use this PR mainly as a bors test tbf